File: pgm/ABC_Java/ABC_366_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String[] arr = reader.readLine().split (" ");
        final String Yes = "Yes";
        final String No = "No";
        final int N, T, A;
        N = Integer.parseInt (arr[0]);
        T = Integer.parseInt (arr[1]);
        A = Integer.parseInt (arr[2]);
        System.out.println (Math.max (T, A) > N / 2 ? Yes : No);
        return;
    }
}

File: pgm/ABC_Visual_Basic/ABC_366_A.vb
' Option Strict On
' Imports System.Collections.Generic
' Imports System.Globalization
' Imports System.IO
' Imports System.Linq
' Imports System.Text
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim arr() As String = Console.ReadLine().Split()
        Const Yes As String = "Yes"
        Const No As String = "No"
        Dim N As Integer = Integer.parse(arr(0))
        Dim T As Integer = Integer.parse(arr(1))
        Dim A As Integer = Integer.parse(arr(2))
        If Math.Max(T, A) > N / 2 Then
            Console.WriteLine(Yes)
        Else
            Console.WriteLine(No)
        End If
        Return
    End Sub
End Class